test/tutorial/test_README_example.cpp

```diff
@@ -30,21 +30,52 @@ int main(const int argc, const char *argv[])
 
   std::vector<Process> reactions{ r1, r2 };
 
-  RosenbrockSolver<> solver{ chemical_system, reactions, RosenbrockSolverParameters::ThreeStageRosenbrockParameters() };
+#if 0
+  auto rosenbrock_solver = Solver::Create()
+                    .SetSystem(chemical_system)
+                    .SetReactions(reactions)
+                    .SetNumberOfGridCells(19)
+                    .Rosenbrock<MatrixPolicy, SparseMatrixPolicy>(RosenbrockSolverParameters::ThreeStageRosenbrockParameters());
```

Comment:
I think we were figuring out how the `SolverBuilder` would work in this test, but I wonder if we should put this somewhere else and keep this test matching what's in the README, so it keeps getting tested?